packages/e2e-tests/fixtures/block-transforms.js

```diff
@@ -303,8 +303,8 @@ export const EXPECTED_TRANSFORMS = {
 		originalBlock: 'Search',
 		availableTransforms: [],
 	},
-	core__section: {
-		originalBlock: 'Section',
+	core__group: {
```

Comment:
Can it be moved to be sorted alphabetically?

Reply:
👍 done